README.md

```diff
@@ -44,7 +40,7 @@ You can also print the table using `getTable` method such as `echo $table->getTa
 
 You can also use `setHeaders()` and `addRow` with Arrays.
 
-    require_once 'ConsoleTable.php';
+    require "vendor/autoload.php";
```

Comment:
Can you please remove this line since it is only valid for composer users? We shall not include `require` in all examples.